### ISSUE TITLE
Use global semantic category mapping via doubly indirect ID mapping from og -> category list

### ISSUE THREAD

**Issue by ChengshuLi** [StanfordVL/OmniGibson#66]
Currently, we do the following to assign semantic labels in base_object.py:

```        
add_update_semantics(
            prim=self._prim,
            semantic_label=self.category,
            type_label="class",
        )
```

However, this probably doesn't preserve globally consistent semantic class, e.g. "apple" is always mapped to value 242.

Solution: omniverse should expose an API to directly set the semantic value. The current API seems to assign semantic ID based on the sequence of inputs, e.g. the first object in the scene will be assigned class id 0.

**Comment by sai-prasanna** [comment]
Hi, do you have any workaround for this? This is really critical for my research where I use semantic segmentation. The ids from the semantic camera changes on each run. Probably due to the change in order in which objects get added. I don't know if there's a way to get the actual class/class id for the ids in the semantic camera observation.

**Comment by cremebrule** [comment]
Right, this is a long-standing issue, and it's currently a limitation from Omniverse.

I have a [feature branch](https://github.com/StanfordVL/OmniGibson/tree/refactor-vision-sensor) that tries to resolve some of these issues, but we still can't directly set IDs, nor get the global IDs yet. We are hoping the next release of IsaacSim will resolve some of these issues.

**Comment by RedCarp0** [comment]
I have noticed that the instance segmentation codes also have some bugs. Although most of the instance labels returned by the seg_instance sensor match the label in helpers.get_instance_mappings(), some labels are incorrect, for example, the "floor" objects are often mislabeled. Have you experienced this issue? 

* Omnigibson version: 0.2.1
* Task: re_shelving_library_books
* Scene: Rs_int

**Comment by hang-yin** [comment]
This should be fixed with #594
